corehq/apps/api/resources/v0_5.py

```diff
@@ -207,6 +208,7 @@ class Meta(v0_1.CommCareUserResource.Meta):
         detail_allowed_methods = ['get', 'put', 'delete']
         list_allowed_methods = ['get', 'post']
         always_return_data = True
+        authentication = RequirePermissionAuthentication(HqPermissions.edit_commcare_users)
```

Comment:
Doesn't this get inherited from the parent class?

Reply:
It does, I meant to delete it when I removed the custom auth